numpy/lib/_npyio_impl.pyi

```diff
@@ -143,26 +143,28 @@ def save(
     arr: ArrayLike,
     allow_pickle: bool = ...,
     *,
-    fix_imports: bool,
+    fix_imports: bool = ...,
 ) -> None: ...
 @overload
 @deprecated("The 'fix_imports' flag is deprecated in NumPy 2.1.")
 def save(
     file: str | os.PathLike[str] | _SupportsWrite[bytes],
     arr: ArrayLike,
-    allow_pickle: bool,
-    fix_imports: bool,
+    allow_pickle: bool = ...,
+    fix_imports: bool = ...,
```

Comment:
The absent defaults for `fix_imports` were intentional here. If not provided, then the first overload will be matched. These last two overloads cover the cases were fix_imports is passed as keyword argument or as positional argument, respectively.
Combined with the `@deprecated` decorator, this has the effect that when `fix_imports`  is given, a deprecation message shows up in the user's IDE or static-type checker.

Reply:
Ah, thanks! There I learn something new.